Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CertificateItem.kt

```diff
@@ -1,5 +1,13 @@
 package de.rki.coronawarnapp.covidcertificate.person.ui.details.items
 
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
 import de.rki.coronawarnapp.util.lists.HasStableId
 
-interface CertificateItem : HasStableId
+interface CertificateItem : HasStableId {
+
+    fun CwaCovidCertificate.toStableListId(): Long = run {
+        var result = this::class.hashCode()
+        result = 31 * result + certificateId.hashCode()
+        result.toLong()
+    }
```

Comment:
Yeah I agree, `containerId` would be better, I'll adjust it.

Reply:
I've changed it to containerId, but had to adjust containerID to not be a data class, otherwise it being a `data class` would override the equals/hashcode and only consider the argument string (which may be the same certificateIds), see:
https://github.com/corona-warn-app/cwa-app-android/pull/3680/commits/1a1e5ad49c8d019624c68f6879bdd33637555a4b